executor/setup.py

```diff
@@ -19,7 +25,7 @@
 setup(
     name="Student API",
     ext_modules = cythonize([
-        Extension("studentapi", sources=sourcefiles, libraries=libraries)
+        Extension("studentapi", sources=sourcefiles, include_dirs=includedirectories, libraries=libraries)

```

Comment:
Why are we adding another parameter, include_dirs here?

Reply:
Ah, yes. So when Cython generates ("cythonizes") the `studentapi` extension (which is how the StudentAPI is currently implemented--perhaps this will change when @BrandonWong14 takes a crack at it later this summer), it needs to know where to find the header files in its C code. Previously, we used quotations to include header files in the student API (i.e. `#include "some_header.h"`) but since i took all of those away and changed them to angle brackets (i.e. `#include <some_header.h>`) for better readability, I had to specify which directories to look for these headers in this line to create the Cython package. 

The official Cython documentation has [this page](https://cython.readthedocs.io/en/latest/src/userguide/source_files_and_compilation.html) describing how it works.